yt_dlp/extractor/lrt.py

```diff
@@ -2,16 +2,61 @@
 from ..utils import (
     clean_html,
     merge_dicts,
+    try_get
 )
 
 
-class LRTIE(InfoExtractor):
-    IE_NAME = 'lrt.lt'
+class LRTStreamIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?lrt\.lt/mediateka/tiesiogiai/(?P<id>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://www.lrt.lt/mediateka/tiesiogiai/lrt-opus',
+        'info_dict': {
+            'id': 'lrt-opus',
+            'live_status': 'is_live',
+            'title': 're:^LRT Opus.+$',
+            'ext': 'mp4'
+        }
+    }]
+
+    def _extract_js_var(self, webpage, var_name, default):
+        return self._search_regex(
+            r'%s\s*=\s*(["\'])((?:(?!\1).)+)\1' % var_name,
+            webpage, var_name.replace('_', ' '), default, group=2)
+
+    def _real_extract(self, url):
+        matches = self._match_valid_url(url).groupdict()
+        video_id = matches['id']
+
+        webpage = self._download_webpage(url, video_id)
+        token_url = self._extract_js_var(webpage, 'tokenURL', None)
+        stream_title = self._extract_js_var(webpage, 'video_title', 'LRT')
+        title = self._og_search_title(webpage)
+
+        streams_data = self._download_json(token_url, video_id)
+
+        formats = []
+        for key in ['content', 'content2', 'content3']:
+            stream_url = try_get(streams_data, lambda x: x['response']['data'][key])
+            if stream_url == '':
+                continue
```

Comment:
If you are trying to get all keys that start with `content`, you can do:
```suggestion
        for stream_url in traverse_obj(streams_data, ('response', 'data', lambda k, _: k.startswith('content')), expected_type=url_or_none):
```
You can also use `in` or `re.match` depending on the exact requirement

PS: feel free to split code into multiple lines as needed